//
// Created by merlinvn@gmail.com on 29/12/16.
//

#include <iostream>
#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
#include "catch/catch.hpp"

//
//int main()  {
//    try {
//        std::cout << "Hello" << std::endl;
//
////        const size_t dimension =1;
//        PersonIndexer<3> pi{Person::Location, Person::AgeClass};
//        pi.init({4,3});
//
//
//        return 0;
//    }
//    catch (const std::exception &e) {
//        std::cout << "Error: " << e.what() << std::endl;
//        return 1;
//    }
//}